src/state_logos.c

```diff
@@ -100,13 +109,25 @@ void state_init_logos(void) {
 }
 
 void state_step_logos(void) {
+#ifdef VERSION_JP
+    int a0 = gGameStatusPtr->pressedButtons[0];
+#endif
+
     if (gGameStatusPtr->bSkipIntro) {
         if (intro_logos_fade_out(10)) {
             set_curtain_scale(1.0f);
             set_curtain_fade(0.0f);
             set_game_mode(GAME_MODE_TITLE_SCREEN);
         }
     } else {
+#ifdef VERSION_JP
+        // TODO: fix fake match
+        if (((u8) gGameStatusPtr->introState - 1 < 0xaU) && (a0 & 0xB000)) {
```

Comment:
this can be accomplished with an or `(introstate == x || introstate == x + 1 ...)`

Reply:
0xB000 should be (0x8000 | 0x2000 | 0x1000) and the values for this are in enums.h (BUTTON_x)